README.md

```diff
@@ -1,2 +1,49 @@
-# unsere-app-fur-die-universitat-regensburg-code-companion
-unsere-app-fur-die-universitat-regensburg-code-companion created by GitHub Classroom
\ No newline at end of file
+# Code Companion
+
+## Concept
+The Goal of the Project is to provide realtime coding assistance on a second screen to help beginners with their coding tasks.  
+This is implemented using an Android App providing real time compiler errors, tasks and coding statistics as well as providing notifications for upcoming project deadlines.
+
+## Implementation
+
+The Project consists of a Plugin for the Jetbrains IntelliJ Platform (plugin marketplace link coming soon) reporting to the connecting Android App (google play link coming soon) using WebRTC.
+This App shows the current tasks, compiler errors and coding statistics.  
+
+
+![](Screenshot.png)  
```

Comment:
Create sections for every fragment. Add a screenshot of the fragment there, and explain in detail what the fragment does exactly